### PR TITLE
Adding first e2e test

### PR DESCRIPTION
**Pull request by jazcarate**
And a bonus health endpoint to make the simplest test possible.

Working towards #43

### CODE REVIEW

File: server/.env.test [hunk 1]
```diff
@@ -0,0 +1,5 @@
+JWT_SECRET=secret-jwt
+MEMORY_DB=true
+REGISTRATION_PASSWORD=password
+SECRET_KEY=secret
+DRIFT_HOME=~/.drift-tests
```
Comment (MaxLeiter) [inline server/.env.test:5]: I wonder if we want to define this in code to the test directory (or a new directory for holding the test db). I can imagine wanting to inspect it and ~/.drift-tests not being intuitive. Thoughts?

Comment (jazcarate) [reply server/.env.test]: To be honest, it is not even needed, as `MEMORY_DB` is set to true 😅. I just forgot to remove it. It is removed now

File: .github/workflows/server-CI.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        working-directory: server
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16'
+      - name: Install deps
+        run: yarn
```
Comment (MaxLeiter) [inline .github/workflows/server-CI.yaml:19]: Do we need to yarn build?

Comment (jazcarate) [reply .github/workflows/server-CI.yaml:19]: Nope. You can check the CI on my fork at: https://github.com/jazcarate/Drift/runs/5804912391?check_suite_focus=true

**Comment by MaxLeiter** [comment]
Thanks @jazcarate! This is great

**Pull request merged**